features/your_contributions_section.feature

```diff
@@ -0,0 +1,25 @@
+Feature:
```

Comment:
What's the description for this feature please? The name of the feature should be the name for this feature file. 

I expect something like this `feature/edit_your_contributions.feature`